src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -73,7 +73,8 @@ object CromwellApiService {
 trait CromwellApiService extends HttpService with PerRequestCreator {
   val workflowManager: ActorRef
 
-  val workflowRoutes = queryRoute ~ workflowOutputsRoute ~ submitRoute ~ callOutputsRoute ~ callStdoutStderrRoute ~ abortRoute
+  val workflowRoutes = queryRoute ~ workflowOutputsRoute ~ submitRoute ~ workflowStdoutStderrRoute ~ abortRoute ~
```

Comment:
I don't have strong feelings about doing it either way, though I'd probably want to check with the primordial source to see if this was a conscious decision made for a compelling reason, or just arbitrary.